x-pack/plugins/search_indices/public/plugin.ts

```diff
@@ -65,8 +65,17 @@ export class SearchIndicesPlugin
     };
   }
 
-  public start(core: CoreStart): SearchIndicesPluginStart {
+  public start(
+    core: CoreStart,
+    deps: SearchIndicesAppPluginStartDependencies
+  ): SearchIndicesPluginStart {
+    const { indexManagement } = deps;
     docLinks.setDocLinks(core.docLinks.links);
+    indexManagement?.extensionsService.setIndexDetailsPageRoute({
+      renderRoute: () => {
+        return '/app/elasticsearch/indices/index_details/';
+      },
+    });
```

Comment:
I think we should take the index name and do the full template here. It would allow us to include a default tab.
```suggestion
      renderRoute: ({ indexName }) => {
        return `/app/elasticsearch/indices/index_details/${indexName}`;
      },
    });
```

Reply:
updated in https://github.com/elastic/kibana/pull/194857/commits/9ab2e9b23640da390cbe2f8dfb7e3e7ced3badf6